src/sentry/taskdemo/__init__.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+
+import logging
+
+from sentry.taskworker.registry import taskregistry
+
+logger = logging.getLogger(__name__)
+demotasks = taskregistry.create_namespace(name="demos")
+
+
+@demotasks.register(name="demos.say_hello")
+def say_hello(name):
```

Comment:
We could put these demo functions in `sentry.taskworker.tasks.examples` 🤷

Reply:
Good idea